infrastructure/src/main/java/org/corfudb/infrastructure/BaseServer.java

```diff
@@ -67,6 +67,43 @@ private void getVersion(CorfuMsg msg, ChannelHandlerContext ctx,
         r.sendResponse(ctx, msg, new JSONPayloadMsg<>(vi, CorfuMsgType.VERSION_RESPONSE));
     }
 
+    boolean checkBootstrap(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r) {
```

Comment:
This probably doesn't need to be here, right?

Reply:
It still needs to check for bootstrap right ? Would it be safe to proceed with incrementing epoch even if the layout server is not bootstrapped?

Reply:
I think it should be safe to increment epoch (what if there is no layout service)?

We just don't want the server to serve a layout before being bootstrapped.... With clusterId that requirement should go away as well

Reply:
I see. I'll make the change